core-bundle/src/Command/InstallCommand.php

```diff
@@ -239,12 +246,24 @@ private function addInitializePhp()
      */
     private function symlinkTcpdfConfig()
     {
+        $relPath = $this->getRelativePath($this->bundlesMeta['ContaoCoreBundle']['path']);
+
         SymlinkUtil::symlink(
-            'vendor/contao/core-bundle/src/Resources/contao/config/tcpdf.php',
+            $relPath.'/Resources/contao/config/tcpdf.php',
             'system/config/tcpdf.php',
             $this->rootDir
         );
 
         $this->io->writeln('Symlinked the <comment>system/config/tcpdf.php</comment> file.');
     }
+
+    /**
+     * @param string $path
+     *
+     * @return string
+     */
+    private function getRelativePath($path)
+    {
+        return str_replace(strtr($this->rootDir, '\\', '/').'/', '', strtr($path, '\\', '/'));
```

Comment:
This should probably use `Filesystem::makePathRelative()`?

Reply:
True.. I relied too much on the original PR 😁. Done now.